modules/bday.js

```diff
@@ -99,42 +138,102 @@ const worker = async ({ command, channel_id, channel_name, response_url }) => {
   // send card to bday user
   if (command.includes('send') && command.includes('to')) {
     const R = /<(?<MIRO_URL>.*)> to <@(?<BDAY_USER_ID>.*)\|(?<BDAY_USER_NAME>.*)>/
+    const RwithMessage = /<(?<MIRO_URL>.*)> to <@(?<BDAY_USER_ID>.*)\|(?<BDAY_USER_NAME>.*)> (?<CUSTOM_MESSAGE>.*)/
     const matches = command.match(R)
+    const matchesWithOptMessage = command.match(RwithMessage)
+
```

Comment:
I feel like these steps are repeated, maybe could do them before checking if it's send|sign since they are the same for both(?)